docker/config/.env.localrunner

```diff
@@ -5,6 +5,12 @@ AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
 AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
 AWS_SESSION_TOKEN=${AWS_SESSION_TOKEN}
 AWS_DEFAULT_REGION=us-west-2
+RDS_PARRSWORD=${RDS_PASSWORD}
+RDS_USER=${RDS_USER}
+RDS_HOST=${RDS_HOST}
+SSH_ADDRESS=${SSH_ADDRESS}
+AIRFLOW_CONN_TEST_POSTGRES_IAM_CONN={"conn_type": "postgres", "schema":"seesaw", "host": "127.0.0.1", "login": "${RDS_USER}", "port": 5432, "password":"${RDS_PASSWORD}", "extra": {"sslmode": "require" }}
```

Comment:
@morrie-seesaw 
schema is set to seesaw in connection. Do we need to change here as well?
I tried with testing schema you provided in slack but with that dag tasks failing.
if we keep seesaw schema here then dag tasks are running fine(other places testing schema used).

Reply:
So for connection we need to use seesaw schema and for running queries we will overwrite from dags. I think this is fine

Reply:
For the SQLExecuteQueryOperator, the schema attribute of the connection is used to set the database connected to. I don't know why this is the case, but passing `database` in the connection throws an error. So `seesaw` here is the database, we then we provide the schema at the script, dag, or task level